test/videojs.ads.test.js

```diff
@@ -936,7 +936,15 @@ QUnit.test('ended event is sent with postroll', function(assert) {
 
   var ended = sinon.spy();
 
```

Comment:
In Phantom you can set ended on a video element, in a real browser it's read-only, so instead I mock the element completely.